### DISCUSSION TITLE
Weekday localization in months

### DISCUSSION THREAD

**Discussion by maxstirliz** [kudrykv/latex-yearly-planner#136] [Q&A] [answered]
Hello everybody! I managed to localize the planner by adding a respective json file. But the weekdays on the moths, i.e. 'M', 'T', 'W', etc. do not change.

As far as I understand this may be done by hardcoding only; probably playing around with locales. Just replacing the letters does not work for me, for [S]aturday and [S]unday do not start with the same letter in my language. By any chance anyone knows how to do that?

**Comment by kudrykv** [comment]
Quick & dirty would be simply to hardcode that for now

**Reply by maxstirliz** [reply]
Is there a single point where I can add a switch statement or changing locale? Sorry for the dumb question.

So far in day.go, I wrapped every instance of d.Time.Weekday in a function that uses a switch statement to return the value needed — it didn't change anything.

**Reply by kudrykv** [reply]
Localization on the main branch is a workaround. Current code does not support that.

If you're talking about little calendar, it's this file that builds corresponding latex: https://github.com/kudrykv/latex-yearly-planner/blob/2dc9ab8cd32e8ad7eaff11ab99c888891022e77f/tpls/monthTabularV2.tpl
and I think it's this atrocity you want to alter: https://github.com/kudrykv/latex-yearly-planner/blob/2dc9ab8cd32e8ad7eaff11ab99c888891022e77f/app/components/cal/day.go#L20-L38

**Reply by maxstirliz** [reply, marked as answer]
Thank you!
Finally resolved the problem.
'name = name[:1]' needs to be tweaked in the WeekHeader function in month.go, so that translate.py does not create new issues.
